app/views/katello/api/v2/subscriptions/base.json.rabl

```diff
@@ -13,3 +13,12 @@ attributes :sockets, :cores, :ram
 attributes :instance_multiplier, :stacking_id, :multi_entitlement
 attributes :type
 attributes :name => :product_name
+attributes :unmapped_guest
+attributes :virt_only
```

Comment:
I think you mean the other way around? Since it is checking explicitly for true, then that piece of UI should be fine. However, the UI shouldn't inherently dictate the API.